MidasCivil_Adapter/CRUD/Create/Elements/Element.cs

```diff
@@ -74,7 +74,7 @@ private bool CreateCollection(IEnumerable<FEMesh> meshes)
 
```

Comment:
We can determine the `index` outside of the `foreach` loop and then pass it by `ref` to reduce the amount of times the text file is read.

`            int index = System.Convert.ToInt32(NextFreeId(typeof(FEMesh)));`